CODE_GENERATION.md

```diff
@@ -55,21 +55,18 @@ resources:
     api_versions:
     - name: "v1beta1"
 ```
-If not overridden in the `generator-config.yaml` file, the default version generated is 
+If not overridden in the `generator-config.yaml` file, the default version generated is
 `v1alpha1`.
 
 When you re-run the generation with this configuration, existing files won't be
-deleted. So you may want to delete everything in `apis/<serviceid>/v1alpha1`and 
+deleted. So you may want to delete everything in `apis/<serviceid>/v1alpha1`and
 then re-run the command.
 
 If `apis/<serviceid>` is created from scratch, please add the new service name to the list called GENERATED_SERVICES in [Makefile](https://github.com/crossplane/provider-aws/blob/master/Makefile#L10).
 
 ### Makefile
 
-```bash
-- GENERATED_SERVICES="apigatewayv2"
-+ GENERATED_SERVICES="apigatewayv2,<serviceid>"
-```
+The serviceids are fetch from the paths of the generator-config.yaml automatically.
 
 If this step fails for some reason, please raise an issue in [code-generator](https://github.com/aws-controllers-k8s/code-generator)
 and mention that you're using Crossplane pipeline.
```

Comment:
Let's remove the whole section as they don't need to know about this at all thanks to your discovery implementation.

Reply:
Done